bnr-utils/nz_azConnector/nz_azConnector.go

```diff
@@ -282,6 +285,14 @@ func (cn *Conn) downloadBkp(outdir string, uniqueid string, blobpath string, str
                     return fmt.Errorf("Error in creating backup directory structure: %v",err)
                 }
 
+                outfile := path.Join(dumpdir, filename)
+                if (strings.HasSuffix(outfile,"locations.txt")){
+                    arrayLoc = append(arrayLoc,outfile)
+                }
+                if (strings.HasSuffix(outfile,"contents.txt")){
+                    arrayContents = append(arrayContents,outfile)
+                }
+

```

Comment:
@nmahalle52 Won't `filename` actually be `locations.txt` or `contents.txt` itself? Shouldn't we do this -

```go
// I would rename 'arrayContents' to 'contents', and 'arrayLoc' to 'locations'
// if this needs to be an array.
switch filename {
case "locations.txt":
    locations = append(locations, path.Join(dumpdir, filename))
case "contents.txt":
    contents = append(contents, path.Join(dumpdir, filename)) 
}
```

Also would there be more than one `locations.txt` and `contents.txt` files? Do we really need it to be a list?

Reply:
Changed list names and if..else to switch..case .

We need list of location and contents because in case of differential backup we will have multiple location.txt and contents.txt.
example:
1/FULL/md/loc1/location.txt
1/FULL/md/contents.txt
2/DIFF/md/loc1/location.txt
2/DIFF/md/contents.txt